src/processor_arm.cpp

```diff
@@ -1890,12 +1890,56 @@ const std::pair<std::string,std::string> &jl_get_llvm_disasm_target(void)
     return res;
 }
 
+#ifndef __clang_gcanalyzer__
 llvm::SmallVector<jl_target_spec_t, 0> jl_get_llvm_clone_targets(void)
 {
-    if (jit_targets.empty())
-        jl_error("JIT targets not initialized");
+
+    auto &cmdline = get_cmdline_targets();
```

Comment:
There seems to be error-prone duplication between this file and `src/processor_x86.cpp`, no?

Reply:
Yes, these files mostly do the same thing, but not really and it's a mess.